libtifiles/trunk/tests/test_tifiles_2.cc

```diff
@@ -97,7 +97,7 @@ static int compare_files(const char *src, const char *dst)
 // Rename file
 static int move_file(const char *oldpath, const char *newpath)
 {
-#if defined(__WIN32__) && !defined(__MINGW32__)
+#if 0 && defined(__WIN32__) && !defined(__MINGW32__)
```

Comment:
I'd say that this is an opportunity to simplify the code, here and below :)

Reply:
Right, I can simplify this whole file a big deal.

Reply:
Done!